src/src/DataStructs/SettingsStruct.cpp

```diff
@@ -543,19 +544,19 @@ bool SettingsStruct_tmpl<N_TASKS>::getSPI_pins(int8_t spi_gpios[3]) const {
   if (isSPI_valid()) {
     # ifdef ESP32
     switch (InitSPI) {
-      case 1:
+      case static_cast<int>(SPI_Options_e::Vspi):
```

Comment:
Why not the other way around, to cast `InitSPI` to `SPI_Options_e` ?
Then the compiler can help you when you may miss one in a switch statement.

```c++
const SPI_Options_e SPI_selection = static_cast<SPI_Options_e>(InitSPI);
```